### DISCUSSION TITLE
Does slint-ui have or will-have support for node-graph UI ?

### DISCUSSION THREAD

**Discussion by nyue** [slint-ui/slint#5514] [Q&A]
Something like this ?

https://qtnodes.readthedocs.io/en/master/overview.html

![calculator](https://github.com/slint-ui/slint/assets/486172/59c6f15c-baf6-471a-8039-2d42d08e99f3)

**Comment by Vadoola** [comment]
There have been some other discussions and possibly even an issue or two about something similar. Right now there is nothing official supported, you would just have to try and built it yourself from scratch. I haven't tried doing it myself, but the most advanced one I saw in the discussions was this one: https://github.com/slint-ui/slint/discussions/5248
